finetune/lm_base.py

```diff
@@ -356,6 +356,12 @@ def _build_model(self, n_updates_total, target_dim, train=True):
             self.valid_writer = tf.summary.FileWriter(self.autosave_path + '/valid', self.sess.graph)
         self.is_built = True
 
+    def _initialize_session(self):
+        gpus = get_available_gpus()
+        os.environ['CUDA_VISIBLE_DEVICES'] = ",".join(gpus)
+        config = tf.ConfigProto(allow_soft_placement=False)
```

Comment:
I think this is probably default anyway. So this line is redundant but there is nothing wrong with being explicit. Your choice. .

Reply:
ah i'll fix that.